fv3core/fv3core/stencils/dyn_core.py

```diff
@@ -594,12 +594,12 @@ def __call__(self, state):
                 )
                 self.riem_solver_c(
                     dt2,
-                    state.cappa,
+                    state.cappa_quantity,
```

Comment:
This change is done mainly to show that the code works, it will be swiftly edited anyways when all of these are passing Quantity objects.